webpack/assets/javascripts/react_app/components/AuditsList/UserDetails.js

```diff
@@ -25,7 +25,7 @@ const UserDetails = ({ isAuditLogin, userInfo, remoteAddress }) => {
           </span>
         </EllipsisWithTooltip>
         <span>
-          <a href={auditPath}>__('Logged-in')</a>
+          <a href={auditPath}>{__('Logged-in')}</a>
```

Comment:
might make sense to create a separate PR for it assuming we want to cherry pick this.

Reply:
https://github.com/theforeman/foreman/pull/6163 created

Reply:
Rebase